data/json/items/generic/music.json

```diff
@@ -11,7 +11,7 @@
   },
   {
     "type": "GENERIC",
-    "id": "cable_instrument",
+    "id": "cable_speaker",
```

Comment:
You should add a migration for this.

Reply:
I'm not renaming the single occurrence, there is another `cable_instrument`, I'm adding a new item by this change.